.github/workflows/pull_request.yml

```diff
@@ -128,8 +128,14 @@ jobs:
       - name: Pytest
         env:
           NEBULA_TEST_LOGS_DIR: ${{ github.workspace }}/build
-        run: ./ci/test.sh test
+        run: ./ci/test.sh test --rm_dir=false
         timeout-minutes: 25
+      - name: Upload logs
+        uses: actions/upload-artifact@v2
+        if: ${{ failure() }}
+        with:	
+          name: nebula-test-logs
```

Comment:
This name should add ${{ matrix.os }} and ${{ matrix.compiler }} to avoid different logs override.

Reply:
Fixed.